docs/install.rst

```diff
@@ -109,48 +109,101 @@ If you make a change to ``media/static-bundles.json``, you'll need to restart Do
 Local Installation
 ------------------
 
```

Comment:
Maybe my own terminology may be off but I consider running docker locally a "local installation". I'd consider this more of a host installation but that's VM speak (host vs image).

Reply:
I guess I see local the way the docs were when we both joined: local == bare metal / directly on the developer's machine. Agreed, Docker is on the local machine but is more a VM-ish situ. 

Anyway, thought I'd leave it as I found it, as that's the terminology the rest of the team seem to run with